web/src/click_handlers.js

```diff
@@ -894,6 +893,35 @@ export function initialize() {
         }
     });
 
+    // COMPOSE BUTTON TOOLTIPS
+
+    const compose_buttons = tippy(
+        ".compose_control_button_container, .compose_control_button[data-tooltip-template-id], .compose_control_button[data-tippy-content]",
```

Comment:
The compose buttons are pretty class-happy, but I wonder if it's worth exploring adding a new shared class for these tooltips? The multiple selectors here look fiddly and brittle to me.

Reply:
I think we should give a try. 
Tried and looks better I guess.